cvmfs/session_context.cc

```diff
@@ -63,7 +63,7 @@ size_t RecvCB(void* buffer, size_t size, size_t nmemb, void* userp) {
 }
 
 SessionContextBase::SessionContextBase()
-    : upload_results_(1000, 1000),
+    : upload_results_(1000000, 1000000),
```

Comment:
These are the increased timeouts, right?  I think we should declare a static constant unsigned in the class for the value.

Reply:
These aren't timeouts, they are max queue sizes, but in the end this change isn't really needed, since we can use large object packs without problem. The number of upload results is in the 10s in this case.

This change dates from when I was debugging the problem, I should have removed it maybe.

Anyway, I can make it a const somewhere.